pangeo_forge_recipes/transforms.py

```diff
@@ -446,25 +445,33 @@ def expand(self, references: beam.PCollection) -> beam.PCollection:
 @dataclass
 class WriteCombinedReference(beam.PTransform, ZarrWriterMixin):
     """Store a singleton PCollection consisting of a ``kerchunk.combine.MultiZarrToZarr`` object.
-
     :param store_name: Name for the Zarr store. It will be created with
         this name under `target_root`.
     :param target_root: Root path the Zarr store will be created inside;
         `store_name` will be appended to this prefix to create a full path.
-    :param output_json_fname: Name to give the output references file. Must end in ``.json``.
+    :param target_chunks: Dictionary mapping dimension names to chunks sizes.
+        If a dimension is a not named, the chunks will be inferred from the data.
+    :param output_file_name: Name to give the output references file.
+    :param output_file_type: Filetype to store reference files as. Options are: "parquet" or "json".
```

Comment:
```suggestion
    :param output_file_name: Name to give the output references file.
    :param output_file_type: Filetype to store reference files as. Options are: "parquet" or "json".
```

Presumably `target_chunks` was not meant to be here?

Also, is it possibly simpler to just have `output_file_name` include a suffix of either `.json` or `.parquet` and infer the file type from the suffix in the body of `write_combined_references`? I see you removed the following below

```python
file_ext = os.path.splitext(output_json_fname)[-1]

if file_ext == ".json":
```

Was there a reason to not just use that? I feel like its friendlier to the user to just specify a single kwarg here.